pkgs/development/libraries/gstreamer/bad/default.nix

```diff
@@ -129,11 +129,6 @@ stdenv.mkDerivation rec {
       src = ./fix-paths.patch;
       inherit (addDriverRunpath) driverLink;
     })
-
-    # vtdec: Use kVTVideoDecoderReferenceMissingErr only when defined
-    # <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/7157>
-    # TODO: Remove this when the build with the newer SDK works.
-    ./darwin-old-sdk-fix.patch
```

Comment:
Forget to commit the deletion?

Reply:
Whoops, fixed!

(Thankfully Jujutsu prevents me from forgetting to `git add`, but I messed it up wrangling commits I think.)